app/femr/business/services/core/IUserService.java

```diff
@@ -116,4 +116,7 @@
      * and/or errors if they exist.
      */
     ServiceResponse<IUser> update(IUser currentUser, Boolean isNewPassword);
+
+    Boolean checkOldPassword(String newpassword,String oldpassword);
+    //Check if the old password is equal to the new password
```

Comment:
If you use the javadoc template (see the comment on the update() method above) then you can click Ctrl+Q in IntelliJ while your cursor is on the method name to see what the method does.